src/Carousel.tsx

```diff
@@ -494,15 +505,15 @@ function CarouselFunc(uncontrolledProps: CarouselProps, ref) {
 
     intervalHandleRef.current = window.setInterval(
       document.visibilityState ? nextWhenVisible : next,
-      interval || undefined,
+      activeChildIntervalRef.current || interval || undefined,
```

Comment:
this will allow `interval`s to be set to 0 (which is useless, but not meaningless)

```suggestion
      activeChildIntervalRef.current ?? interval ?? undefined,
```